src/types/types.ts

```diff
@@ -21,11 +20,23 @@ export type Props = {
 
 export type AppInputType = {
   onChangeText: (text: string) => void;
+  onBlur: (e: FocusEvent) => void;
   value: string;
   label: string;
+  error: boolean | undefined;
+  errorMessage?: string | undefined;
```

Comment:
make `error` and `onBlur` prop optional, remove` | undefined` from `error` and `errorMessage`